src/Task/ComposerNormalize.php

```diff
@@ -48,6 +49,11 @@ public function run(ContextInterface $context): TaskResultInterface
 
         $arguments = $this->processBuilder->createArgumentsForCommand('composer');
```

Comment:
If you use standalone and "composer" could not be found in your path, this crashes.
Better to call the `createArgumentsForCommand` method with the real tool name based on the `use_standalone` param